test/test_gemm.cc

```diff
@@ -74,6 +77,15 @@ void test_gemm_work(Params& params, bool run)
     if (! run)
         return;
 
+    #ifndef SLATE_HAVE_SCALAPACK
+        // Can only run ref when we have ScaLAPACK
+        if (ref) {
+            if (mpi_rank == 0)
+                printf( "ScaLAPACK not available\n" );
```

Comment:
Perhaps:
```
params.msg() = "ScaLAPACK not available";
```

Reply:
I see — allocate `C_alloc.ref` only if ref && SLATE_HAVE_SCALAPACK.

Reply:
I ultimately switched to putting it in `params.msg()` in #157 : https://github.com/icl-utk-edu/slate/pull/157/files#diff-d278bb6ad93b3c28dbc626301e5ed81edbf1e2391f3b4e6ac37dde93dda1f189R51